packages/integration-karma/scripts/karma-configs/base.js

```diff
@@ -52,7 +45,7 @@ function getFiles() {
         frameworkFiles.push(createPattern(LWC_ENGINE_COMPAT));
         frameworkFiles.push(createPattern(WIRE_SERVICE_COMPAT));
     } else {
-        if (!DISABLE_SYNTHETIC || FORCED_NATIVE_SHADOW_MODE_FOR_TEST) {
+        if (SYNTHETIC_SHADOW_ENABLED) {
```

Comment:
Should this include `|| FORCED_NATIVE_SHADOW_MODE_FOR_TEST` so that synthetic shadow is loaded in that mode too?

Reply:
Synthetic shadow is already toggleable via `DISABLE_SYNTHETIC`. We should avoid overloading configs and mixing concerns.